src/particle/particle/particle.py

```diff
@@ -1015,6 +1015,76 @@ def from_evtgen_name(cls: type[Self], name: str) -> Self:
         """
         return cls.from_pdgid(EvtGenName2PDGIDBiMap[name])
 
+    @classmethod
+    def from_nucleus_info(
+        cls: type[Self],
+        z: int,
+        a: int,
+        anti: bool = False,
+        i: int = 0,
+        l_strange: int = 0,
+    ) -> Self:
+        """
+        Get a nucleus particle from the proton Z and atomic mass A numbers.
+        As described in the PDG numbering scheme:
+        "To avoid ambiguities, nucleus codes should not be applied to a single hadron, such as the p, n
+        or the Λ, where quark-contents-based codes already exist."
+        Number of neutrons is equal to a-z.
+        PDGid format is ±10LZZZAAAI (see https://github.com/scikit-hep/particle/blob/master/src/particle/pdgid/functions.py#L232).
```

Comment:
```suggestion
        The PDG ID format is ±10LZZZAAAI (see https://github.com/scikit-hep/particle/blob/master/src/particle/pdgid/functions.py#L232).
```

Reply:
Update this way because the line number is bound to change as soon as the file gets updates ;-).